src/ripple/rpc/handlers/AccountOffers.cpp

```diff
@@ -24,6 +24,19 @@
 
 namespace ripple {
 
+void appendOfferJson (std::shared_ptr<SLE const> const& offer,
```

Comment:
In many previous codebases I used, there were a fairly small number of classes that were used a great deal that had very short names - which would include STL things e.g. 

    template <class C> 
    using shared = std::shared_ptr<C>;

    using int32 = std::int32_t;

Usually there were so few of these short ones that they fit on a single page, so every programmer knew all of them.  This is the page you show a .cpp programmer on their first day.

Then there were a hundred or so "fairly common" names that had "moderately short" names.

And everything else used the long, explicit names, because they were only used a few times or within one file.